clang-tidy/action.yml

```diff
@@ -0,0 +1,87 @@
+name: "clang-tidy"
+description: "Run clang-tidy on the source files"
+
+inputs:
+  rosdistro:
+    description: "ROS distro"
+    required: true
+  build-depends-repos:
+    description: "VCS repositories containing build dependencies"
+    required: true
+  target-files:
+    description: "Target files to run clang-tidy on"
+    required: true
+  compile-commands-hash:
+    description: Calculated hash of compile_commands.json
+    required: true
+  github-access-token:
+    description: "If the repository is private, specify a token."
+    required: false
+
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout repository
+      if: ${{ inputs.target-files != '' }}
```

Comment:
same as https://github.com/autowarefoundation/autoware-github-actions/pull/4#discussion_r763749610

Reply:
fa09d64

Reply:
Thank you! Then please add the empty check here instead.

https://github.com/autowarefoundation/autoware.universe/pull/104/files#diff-d549e0b4270f5b02caec3d0bf592280fe38c24737b9d34cc9cbee0552ab0773aR20-R26

Reply:
Added https://github.com/autowarefoundation/autoware.universe/pull/104/commits/0759fb96e1752e4c9da52f3de27e63434628fed3

Reply:
Does no `!= ''` work correctly? I'm not sure.
Could you refer to the official document (of context variables?) if possible?

Reply:
Empty strings will be converted to 0, so I think it will work.
https://docs.github.com/en/actions/learn-github-actions/expressions#operators